runtime/near-vm-runner-standalone/src/main.rs

```diff
@@ -165,14 +158,12 @@ fn main() {
     let mut results = script.run();
     let (outcome, err) = results.outcomes.pop().unwrap();
 
-    println!(
```

Comment:
Let's keep this as a `printlin` -- this is the primary output and result of the tool.